esque/controller/consumergroup_controller.py

```diff
@@ -93,15 +93,15 @@ def create_consumer_group_offset_change_plan(
             self._logger.error("The consumer group {} does not exist.".format(consumer_id))
             return None
         elif consumer_group_state == "Empty":
-            if offset_to_value:
+            if offset_to_value is not None:
```

Comment:
What. That shouldn’t change anything?

Reply:
Yes it does because `0` is a valid value but doesn't evaluate to `True`. That's why we have to check if it's `None` instead.

Reply:
gotcha. thanks for the clarification.